packages/ui/src/composables/useSelectableList.ts

```diff
@@ -9,8 +9,8 @@ type StringOrFunction = string | ((option: SelectableOption) => unknown)
 export const useSelectableListProps = {
   options: { type: Array as PropType<SelectableOption[]>, default: () => [] },
   textBy: { type: [String, Function] as PropType<StringOrFunction>, default: 'text' },
-  valueBy: { type: [String, Function] as PropType<StringOrFunction>, default: '' },
-  trackBy: { type: [String, Function] as PropType<StringOrFunction>, default: 'value' },
+  valueBy: { type: [String, Function] as PropType<StringOrFunction>, default: 'value' },
```

Comment:
```suggestion
  valueBy: { type: [String, Function] as PropType<StringOrFunction>, default: '' },
```

Let's better have it as empty string. Because there is no way to get object as value, user will need to pass `:value-by="''"`